uber/site_sections/jobs.py

```diff
@@ -170,7 +170,7 @@ def staffers_by_job(self, session, id, message=''):
         return {
             'job':       job,
             'message':   message,
```

Comment:
this is commenting on an outdated line so I'm going to delete this comment, and re-annotate what is going on here